Automattic/sensei
# Add answer feedback editor settings

Fixes #4338 

### Changes proposed in this Pull Request

* Add  controls for new answer feedback settings to the quiz block in the editor
* Create new meta on the quiz post for _failed_show_answer_feedback, _failed_show_correct_answers, _failed_indicate_incorrect, and add them to the `lesson-quiz` REST API endpoint

### Testing instructions

* Edit a lesson with a quiz block
* Toggle on 'Pass required' in the quiz block settings
* `Indicate which questions are incorrect.`, `Show correct answers.`, `Show “Answer Feedback” text.` options should appear
* Make sure changing these settings is preserved upon saving & opening the lesson editor again

### Screenshot / Video

![image](https://user-images.githubusercontent.com/176949/137129483-107e1fc7-7cdb-4050-9bfa-45e918cd3350.png)
